infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/KubernetesPluginsToolingApplier.java

```diff
@@ -194,6 +196,7 @@ private void addSidecar(
             .build();
 
     InternalMachineConfig machineConfig = machineResolver.resolve();
+    machineConfig.getAttributes().put(CONTAINER_SOURCE_ATTRIBUTE, TOOL_CONTAINER_SOURCE);
```

Comment:
It would be nice to have the corresponding test in `KubernetesPluginsToolingApplierTest`